app/plan.go

```diff
@@ -51,7 +53,32 @@ func (s *planService) FindByName(name string) (*appTypes.Plan, error) {
 
 // DefaultPlan implements DefaultPlan method of PlanService interface
 func (s *planService) DefaultPlan() (*appTypes.Plan, error) {
-	return s.storage.FindDefault()
+	plan, err := s.storage.FindDefault()
+	if err != nil && err == appTypes.ErrPlanDefaultNotFound {
+		plans, err := s.storage.FindAll()
+		if err != nil {
+			log.Errorf("Error listing Plans: %s", err)
+			return nil, appTypes.ErrPlanDefaultNotFound
```

Comment:
I think it would be better to return the original error here, returning plan not found may mislead the end user.

Reply:
But it's returning the original error ("default plan not found")